docs/Makefile

```diff
@@ -12,7 +12,7 @@ build-helm:
 	$(MAKE) -C ../helm build
 
 reference/attributions.txt: $(shell find ../tools/scripts/licenses -type f)
-	../tools/scripts/gen-attributions.py sphinx > $@
+	python ../tools/scripts/gen-attributions.py sphinx > $@
```

Comment:
Why this change?

Reply:
ah this snuck it, was having issues building docs